components/greeting/Greeting.tsx

```diff
@@ -3,8 +3,8 @@ import styles from "./Greeting.module.css";
 function Greeting() {
   return (
     <div className={styles.container}>
-      <h1>Welcome to ****</h1>
-      <h2>A Niffler Fangame</h2>
+      <h1 className={styles.heading}>Welcome to ****</h1>
+      <h2 className={styles.subheading}>A Niffler Fangame</h2>
```

Comment:
Maybe use Camel Case here "subHeading".

Reply:
Since subheading is an own word, I prefer not to change it. As soon as two words come together I'm a huge fan of Camel Case :)